### PR TITLE
Lizard Virus

### PR DESCRIPTION
**Pull request by Profakos**
Made by Collen, fixed by me. Turns the afflicted person into a lizard.
Adds a bottle too, for shenanigans.

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/lizardification.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+					H << "<span class='notice'>[pick("Your skin itches.", "You feel a tingling underneath your skin.", "You feel goosebumps on your skin.")]</span>"
+				if(3, 4)
+					H << "<span class='notice'>[pick("Your nose seems to elongate.", "Scales start pushing out of your skin.", "Your teeth feel sharper.")]</span>"
+				if(5)
+					if(H.dna && !H.dna.mutantrace)
+						H  << "<span class='danger'>You feel coldblooded.</span>"
+						H.dna.mutantrace = "lizard"
+						H.update_body()
+						H.update_hair()
+						A.cure()
```
Comment (Giacom) [inline code/datums/diseases/advance/symptoms/lizardification.dm]: That's going to cure them of all their other symptoms; just have it so you check if they're a lizard or not in the 5th stage after they are turned.

Another possible feature, that you can add, is that when the symptom dies you can use the symptom proc "end()" to reverse the lizard form.

**Comment by Profakos** [comment]
Collen said that the first bit is intentional, once the virus did its job, it goes away completely. However, moving the selfcure part out of the fifth stage is a good idea.

On the other hand, she did it this way because neither I or she knew how to cure results of the symptons when the virus goes away. Thank you for the help! I'll replace self-cure with that.

**Comment by Profakos** [comment]
Again, thank you! Now we know how to make viruses that heal a lasting change they caused.

**Comment by Giacom** [comment]
You should check they aren't a lizard already before displaying the symptom messages.

**Comment by Profakos** [comment]
Alright, moved the check before the message display. Now lizards won't try to turn into lizards (neither will other mutanttraces).

**Pull request merged**